Tests/SentryTests/Helper/SentryExtraContextProviderTests.swift

```diff
@@ -64,11 +64,14 @@ final class SentryExtraContextProviderTests: XCTestCase {
     func testExtraProcessInfo() {
         let sut = fixture.getSut()
         fixture.processWrapper.overrides.processorCount = 12
-        
+        fixture.processWrapper.overrides.thermalState = .critical
+
         let actualContext = sut.getExtraContext()
         let device = actualContext["device"] as? [String: Any]
         
         XCTAssertEqual(device?["processor_count"] as? UInt, fixture.processWrapper.overrides.processorCount)
+        XCTAssertEqual(device?["thermal_state"] as? ProcessInfo.ThermalState,
```

Comment:
Needed to extract this as a String, not a thermal state enum value.
```suggestion
        XCTAssertEqual(device?["thermal_state"] as? ProcessInfo.ThermalState,
```